### PR TITLE
[AzureMonitorDistro] new project: Azure.Monitor.OpenTelemetry

### PR DESCRIPTION
**Pull request by TimothyMothra**
Creating a new directory/solution/project for Azure.Monitor.OpenTelemetry.Distro

Related to https://github.com/Azure/azure-sdk/issues/5423

# Contributing to the Azure SDK

Please see our [CONTRIBUTING.md](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md) if you are not familiar with contributing to this repository or have questions.

For specific information about pull request etiquette and best practices, see [this section](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#pull-request-etiquette-and-best-practices).

### CODE REVIEW

File: sdk/monitor/Azure.Monitor.OpenTelemetry.Distro/Azure.Monitor.OpenTelemetry.Distro/Azure.Monitor.OpenTelemetry.Distro.csproj
Comment (rajkumar-rangaraj) [inline sdk/monitor/Azure.Monitor.OpenTelemetry.Distro/Azure.Monitor.OpenTelemetry.Distro/Azure.Monitor.OpenTelemetry.Distro.csproj:6]: Add distro here.

**Comment by rajkumar-rangaraj** [comment]
Update https://github.com/Azure/azure-sdk-for-net/blob/main/.github/CODEOWNERS to include the same owners as of exporters.

**Comment by rajkumar-rangaraj** [comment]
@nisha-bhatia Could you please confirm if the project format is good for merge.

**Review by lmolkova — COMMENTED** [review]
AFAIK the naming pattern is going to be Azure.Monitor.ApplicationInsights.OpenTelemetry. We should have more clarity after Thursday arch board though.

You probably also want to add CI for distro here https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/monitor/ci.yml

File: sdk/monitor/Azure.Monitor.OpenTelemetry.Distro/Azure.Monitor.OpenTelemetry.Distro/Azure.Monitor.OpenTelemetry.Distro.csproj
Comment (TimothyMothra) [reply sdk/monitor/Azure.Monitor.OpenTelemetry.Distro/Azure.Monitor.OpenTelemetry.Distro/Azure.Monitor.OpenTelemetry.Distro.csproj:6]: Do we want both Exporter and Distro here or only Distro?

**Comment by TimothyMothra** [comment]
> AFAIK the naming pattern is going to be Azure.Monitor.ApplicationInsights.OpenTelemetry. We should have more clarity after Thursday arch board though.
> 
> You probably also want to add CI for distro here https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/monitor/ci.yml

@lmolkova what's the process to make changes to the ci.yml file?

https://github.com/Azure/azure-sdk-for-net/blob/a643c394111e463194b33b7648579b9ceb4fcc24/sdk/monitor/ci.yml#L1-L2

File: .github/CODEOWNERS
Comment (TimothyMothra) [inline .github/CODEOWNERS:485]: I assume we'll want to use the same "Exporter" labels because this is the same team and essentially the same product. 

Please comment if this needs to be different.

File: sdk/monitor/Azure.Monitor.ApplicationInsights.OpenTelemetry/Azure.Monitor.ApplicationInsights.OpenTelemetry/Azure.Monitor.ApplicationInsights.OpenTelemetry.csproj
Comment (rajkumar-rangaraj) [inline sdk/monitor/Azure.Monitor.ApplicationInsights.OpenTelemetry/Azure.Monitor.ApplicationInsights.OpenTelemetry/Azure.Monitor.ApplicationInsights.OpenTelemetry.csproj:12]: Azure.Monitor.OpenTelemetry.Exporter.csproj has indirect reference to `Azure.Core`. We don't need to add here again.

**Comment by TimothyMothra** [comment]
Getting the following ci error:

`eng\service.proj(43,5): Error : No Projects found with patttern [..\sdk\monitor/Azure.Monitor.ApplicationInsights.OpenTelemetry\**\*.csproj], please make sure you have passed in the correct ServiceDirectory.`

Clearly I've missed a step but not sure where. :)
This is the new project's full path:

"C:\REPOS\AzureSDK\sdk\monitor\Azure.Monitor.ApplicationInsights.OpenTelemetry\Azure.Monitor.ApplicationInsights.OpenTelemetry\Azure.Monitor.ApplicationInsights.OpenTelemetry.csproj"

File: sdk/monitor/Azure.Monitor.ApplicationInsights.OpenTelemetry/Azure.Monitor.ApplicationInsights.OpenTelemetry/Azure.Monitor.ApplicationInsights.OpenTelemetry.csproj
Comment (TimothyMothra) [reply sdk/monitor/Azure.Monitor.ApplicationInsights.OpenTelemetry/Azure.Monitor.ApplicationInsights.OpenTelemetry/Azure.Monitor.ApplicationInsights.OpenTelemetry.csproj:12]: good catch. i'll remove this now

File: sdk/monitor/Azure.Monitor.ApplicationInsights.OpenTelemetry/Azure.Monitor.ApplicationInsights.OpenTelemetry/Azure.Monitor.ApplicationInsights.OpenTelemetry.csproj
Comment (rajkumar-rangaraj) [inline sdk/monitor/Azure.Monitor.ApplicationInsights.OpenTelemetry/Azure.Monitor.ApplicationInsights.OpenTelemetry/Azure.Monitor.ApplicationInsights.OpenTelemetry.csproj:1]: This project should be created inside a `/src` directory. 

here - sdk/monitor/Azure.Monitor.ApplicationInsights.OpenTelemetry/Azure.Monitor.ApplicationInsights.OpenTelemetry/src/Azure.Monitor.ApplicationInsights.OpenTelemetry.csproj

File: .github/CODEOWNERS
Comment (rajkumar-rangaraj) [reply .github/CODEOWNERS:485]: Let us have a different label name. It is easier for maintenance.

File: sdk/monitor/Azure.Monitor.ApplicationInsights.OpenTelemetry/Azure.Monitor.ApplicationInsights.OpenTelemetry/Azure.Monitor.ApplicationInsights.OpenTelemetry.csproj
Comment (TimothyMothra) [reply sdk/monitor/Azure.Monitor.ApplicationInsights.OpenTelemetry/Azure.Monitor.ApplicationInsights.OpenTelemetry/Azure.Monitor.ApplicationInsights.OpenTelemetry.csproj:1]: oh i see. will make that change now

File: .github/CODEOWNERS
Comment (TimothyMothra) [reply .github/CODEOWNERS:485]: I named the new label "Monitor - Distro" for now.

File: sdk/monitor/Azure.Monitor.ApplicationInsights.OpenTelemetry/src/Azure.Monitor.ApplicationInsights.OpenTelemetry.csproj
Comment (rajkumar-rangaraj) [inline sdk/monitor/Azure.Monitor.ApplicationInsights.OpenTelemetry/src/Azure.Monitor.ApplicationInsights.OpenTelemetry.csproj:5]: Looks like CI is not happy with alpha version, may be changing to beta could help.

**Review by vishweshbankwar — APPROVED** [review]

**Review by lmolkova — APPROVED** [review]
We didn't reach an agreement on the package name,  but it shouldn't be blocking  - hopefully we will before first beta of this package.

**Review by rajkumar-rangaraj — APPROVED** [review]

**Comment by TimothyMothra** [comment]
> We didn't reach an agreement on the package name, but it shouldn't be blocking - hopefully we will before first beta of this package.

To unblock ourselves, we can do what Ted suggested and publish to the dev feed until the name is finalized. :)

File: sdk/monitor/Azure.Monitor.ApplicationInsights.OpenTelemetry/README.md
Comment (reyang) [inline sdk/monitor/Azure.Monitor.ApplicationInsights.OpenTelemetry/README.md:29]: Is this the actual intention?

```suggestion
The library is not yet _generally available_, and is not officially supported. Future releases will not attempt to maintain backwards compatibility with previous releases. Each beta release includes significant changes to the distro package, making them incompatible with each other.
```

File: .github/CODEOWNERS
Comment (jsquire) [inline .github/CODEOWNERS:488]: For this to work,  we'll also need the PR path and Service Attention rules to be created in [fabricbot.json](https://github.com/Azure/azure-sdk-for-net/blob/main/.github/fabricbot.json#L2273) and the "Monitor -Distro" label created in the repository with color `#e99695`.   _(the color is significant to automation)_

**Review by cijothomas — APPROVED** [review]

File: .github/CODEOWNERS
Comment (TimothyMothra) [reply .github/CODEOWNERS:488]: I created a new label "Monitor - Distro" and assigned to this PR.

Comment (TimothyMothra) [reply .github/CODEOWNERS:488]: @jsquire, I updated the fabricbot. Please review

**Review by reyang — APPROVED** [review]

**Comment by reyang** [comment]
LGTM with the assumption that this is a temporary placeholder to unblock the dev work.

**Review by rajkumar-rangaraj — APPROVED** [review]
`Azure.Monitor.OpenTelemetry` is a lot simpler. LGTM.

**Review by vishweshbankwar — APPROVED** [review]

**Pull request merged**
